docs/user_guide.md

```diff
@@ -165,8 +166,21 @@ The system administrator can also operate the replication rules defined for the
 
 ![browse project](img/rule_under_project_view.png)
 
+## Retag Images
 
-## Searching projects and repositories  
+Images retag helps users to tag images in Harbor, images can be tagged to  different repositories and projects, as long as the users have sufficient permissions. For example,
+
+```
+release/app:stg  -->  release/app:prd
+develop/app:v1.0 --> release/app:v1.0
+```
+To retag an image, users should have read permission (guest role or above) to the source project and write permission (developer role or above) to the target project.
+
+In Harbor portal, select the image you'd like to retag, and click the enabled `Retag` button to open the retag dialog.
+
+![retag image](img/retag_image.png)
+

```

Comment:
Is it possible to add one more sentence to describe what user should do after opening the dialog? It looks like a broken process, abruptly ended.

ps: I did not see the screenshot picture, could u please check if the png is correctly uploaded?